SofaKernel/framework/sofa/helper/ArgumentParser.cpp

```diff
@@ -105,16 +108,20 @@ void ArgumentParser::operator () ( std::list<std::string> str )
         // display help
         if( name == shHelp || name == lgHelp )
         {
-            if( globalHelp.size()>0 )
-                std::cout<< globalHelp <<std::endl;
-
+            if( globalHelp.size()>0 ) std::cout<< globalHelp <<std::endl;
+            
             std::cout << "(short name, long name, description, default value)\n-h,\t--help: this help" << std::endl;
             std::cout << std::boolalpha;
             for( ArgVec::const_iterator a=commands.begin(), aend=commands.end(); a!=aend; ++a )
                 (*a)->print();
             std::cout << std::noboolalpha;
+
+            std::cout << "--argv [...]\t" << "forward extra args to the python interpreter" << std::endl;
```

Comment:
The PR started from discussion in #356, so I sticked to that. I don't quite see where else it could be useful but if needed, extra arguments can be accessed via the static function `ArgumentParser::extra_args()`